From a8ac756000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: DymondZ <gh3ttokill4@gmail.com>
Date: Fri, 19 Jun 2026 22:30:00 +0200
Subject: [PATCH 2/5] sweet: audio: Add ACDB IDs for voice recording

Define calibration IDs for the AEC and NS voice-recording input devices.
---
 configs/audio/audio_platform_info_intcodec.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configs/audio/audio_platform_info_intcodec.xml b/configs/audio/audio_platform_info_intcodec.xml
--- a/configs/audio/audio_platform_info_intcodec.xml
+++ b/configs/audio/audio_platform_info_intcodec.xml
@@ -95,6 +95,8 @@
         <device name="SND_DEVICE_OUT_SPEAKER_AND_HEADPHONES_EXTERNAL_1" acdb_id="10"/>
         <device name="SND_DEVICE_OUT_SPEAKER_AND_HEADPHONES_EXTERNAL_2" acdb_id="10"/>
         <device name="SND_DEVICE_OUT_SPEAKER_AND_USB_HEADSET" acdb_id="45"/>
+        <device name="SND_DEVICE_IN_VOICE_REC_MIC_AEC" acdb_id="111"/>
+        <device name="SND_DEVICE_IN_VOICE_REC_MIC_NS" acdb_id="111"/>
         <device name="SND_DEVICE_IN_UNPROCESSED_USB_HEADSET_MIC" acdb_id="133"/>
         <device name="SND_DEVICE_IN_UNPROCESSED_MIC" acdb_id="143"/>
         <device name="SND_DEVICE_IN_UNPROCESSED_STEREO_MIC" acdb_id="144"/>
-- 
2.53.0
